tsconfig.node.json

```diff
@@ -4,8 +4,10 @@
     "skipLibCheck": true,
     "module": "ESNext",
     "moduleResolution": "bundler",
+    "allowSyntheticDefaultImports": true,
     "jsx": "react-jsx",
-    "allowSyntheticDefaultImports": true
+    "noEmit": true,
```

Comment:
why did u add noEmit ?

Reply:
imports with .tsx extensions were showing typescript error.
![image](https://github.com/raj-kumar-litmus/support-hub/assets/146818733/f596e7bd-c92a-4251-9c26-009660a1c2e7)

Reply:
Deployment is failing because of noEmit.
If you Remove that line, are you able to  commit it ?
<img width="1248" alt="Screenshot 2023-10-26 at 4 51 53 PM" src="https://github.com/raj-kumar-litmus/support-hub/assets/132570273/f3748690-75d1-4e33-9061-ca8eef8fa7fe">

Reply:
Removed noEmit, working fine after removing .tsx extension on imports